calicoctl/commands/node/run.go

```diff
@@ -69,6 +69,7 @@ func Run(args []string) error {
                      [--no-default-ippools]
                      [--dryrun]
                      [--init-system]
+					 [--allow-version-mismatch]
```

Comment:
Looks like there's a spacing error here, probably need to change it from tabs to spaces?

Reply:
Oh, I missed it in my editor, thanks for catching it, will fix!